packages/bun-types/test.d.ts

```diff
@@ -21,6 +21,8 @@ declare module "bun:test" {
    */
   export type Mock<T extends (...args: any[]) => any> = JestMock.Mock<T>;
 
+  export const fail: (message?: string) => void;
+
```

Comment:
probably could use `never` over `void` because it should effectively throw an error (the execution of below lines wont happen)

https://www.typescriptlang.org/docs/handbook/2/functions.html#never
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/jest/index.d.ts#L1568